Automattic/jetpack
# CRM: Add the order capability to the API events endpoint.

Fixes https://github.com/Automattic/zero-bs-crm/issues/3476

## Proposed changes:

This PR adds the result `order` capability to the `events` endpoint.

Example of a request:

- http://jetpackcrm.local/zbs_api/events/?api_key=jpcrm_pk_e75f7369e0e92cf88c323963fbf265d2611b2139&api_secret=jpcrm_sk_b6095358cfd10c7f85e1cae3be0bce0436b828f4&page=1&perpage=2&order=ASC
- http://jetpackcrm.local/zbs_api/events/?api_key=jpcrm_pk_e75f7369e0e92cf88c323963fbf265d2611b2139&api_secret=jpcrm_sk_b6095358cfd10c7f85e1cae3be0bce0436b828f4&page=1&perpage=2&order=DESC

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
- Check the code
- Activate the API module
- Configure your API secret keys
- With some already added events (schedule item), call the API endpoint with these parameters
   - page: 1
   - perpage: 1
   - order: ASC and DESC
- You will see the result in different order
